### ISSUE TITLE
Symbol lookup error, undefined symbol: alpm_option_set_arch

### ISSUE THREAD

**Issue by Dartegnian** [Morganamilo/paru#361]
### Affected Version
`paru -V`: Doesn't even work, but is version `1.6.0.r12.g04fa8dc-1`
`pacman -V`: `6.0.0alpha1-107-gef14c`

### Description
**Have you checked previous issues for this bug?**
Yes, no similar tickets found in the `paru` repo, but exists in the [`yay`](https://github.com/Jguer/yay/issues/1497) repo. Could [this](https://git.archlinux.org/pacman.git/commit/?id=3179db108a83104d9de6d1d607f55f8118e92160) be affecting paru?

### Output
Include the FULL output of any relevant commands/configs

```sh
$ paru -V
paru: symbol lookup error: paru: undefined symbol: alpm_option_set_arch

$ paru -S pacman
paru: symbol lookup error: paru: undefined symbol: alpm_option_set_arch

$ paru -R neofetch
paru: symbol lookup error: paru: undefined symbol: alpm_option_set_arch

$ paru --help
paru: symbol lookup error: paru: undefined symbol: alpm_option_set_arch
```

**Comment by Morganamilo** [comment]
Paru needs to be built against the version of pacman you're using. Just rebuild paru.

**Comment by trevorgunn** [comment]
@Morganamilo should you mark the `paru-bin` package as incompatible with `pacman>=6`?

**Comment by Dartegnian** [comment]
> Paru needs to be built against the version of pacman you're using. Just rebuild paru.

Yup, rebuilding helped. Still unsure of what caused the bug, but I'm good with the fix provided. Thanks!
